### ISSUE TITLE
Enum parse error

### ISSUE THREAD

**Issue by DigitalFeonix** [softwaretechnik-berlin/dbml-renderer#11]
If you have an `Enum` with a value that requires putting it in quotes -- because of a space, parenthesis, or other special character -- the renderer fails to parse the DBML.

test.dbml
```
Enum order_status {
    "Abandoned Cart"
    Submitted
    Paid
    Shipped
    Delivered
}

Table orders {
    id int [pk, increment]
    status order_status
}
```

```shell
> dbml-renderer -i test.dbml -o test.svg
Could not parse input at line 2. Expected "//", "}", or [a-zA-Z_] but "\"" found.
```

While the [DBML docs](https://www.dbml.org/docs/#enum-definition) do not show any example with a space in the value, testing on [dbdiagram.io](https://dbdiagram.io/d) shows it does work -- and that only double quotes work.

![image](https://user-images.githubusercontent.com/197201/154140838-2d0b9bf5-320b-4788-9de1-8bb0224a2b51.png)

**Comment by DigitalFeonix** [comment]
I also found that quotes around an Enum name causes a parse error.

test.dbml
```
Enum "dbml_test_string_enum" {
  Choice1
  Choice2
  Choice3
}
```

```shell
> dbml-renderer -i test.dbml -o test.svg
Could not parse input at line 1. Expected [a-zA-Z_] but "\"" found.
```

And if you have non Alpha character start an Enum value it fails as well

test.dbml
```
Enum dbml_test_number_enum {
  1
  2
  3
}
```

```shell
> dbml-renderer -i test.dbml -o test.svg
Could not parse input at line 2. Expected "//", "}", or [a-zA-Z_] but "1" found.
```

**Comment by barnardb** [comment]
Thanks for reporting. The examples you've given in this issue and the follow-up comment have been added as a test case, and it's been fixed in v1.0.20.
